src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 0 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  border: 1px solid $color-close-to-white;
+  border-radius: 5px;
```

Comment:
You should create a variable for the border-radius because we use it more than one time.

Reply:
agree